buildspec/linuxUiTests.yml

```diff
@@ -33,16 +34,25 @@ phases:
         ffmpeg -f x11grab -video_size 1920x1080 -i :99 -codec:v libx264 -r 12 /tmp/screen_recording.mp4 &
         fi
       - ./gradlew guiTest coverageReport --info --console plain
+
   post_build:
     commands:
+      - TEST_ARTIFACTS="/tmp/testArtifacts"
+      - mkdir $TEST_ARTIFACTS
+      - rsync -r guitest.log $TEST_ARTIFACTS/gui/ || true
+      - rsync -rmv --include='*/' --include '**/build/idea-sandbox/system*/log/**' --exclude='*' . $TEST_ARTIFACTS/ || true
+      - rsync -rmv --include='*/' --include '**/build/reports/**' --exclude='*' . $TEST_ARTIFACTS/ || true
+
       - if [ "$RECORD_UI" ]; then pkill -2 ffmpeg; while pgrep ffmpeg > /dev/null; do sleep 1; done; fi
-      - if [ "$RECORD_UI" ]; then cp /tmp/screen_recording.mp4 jetbrains-core-gui/build/idea-sandbox/system/log/screen_recording.mp4; fi
+      - if [ "$RECORD_UI" ]; then cp /tmp/screen_recording.mp4 $TEST_ARTIFACTS/gui/; fi
+
       - VCS_COMMIT_ID="${CODEBUILD_RESOLVED_SOURCE_VERSION}"
       - CI_BUILD_URL=$(echo $CODEBUILD_BUILD_URL | sed 's/#/%23/g') # Encode `#` in the URL because otherwise the url is clipped in the Codecov.io site
       - CI_BUILD_ID="${CODEBUILD_BUILD_ID}"
       - curl -s https://codecov.io/bash > codecov.sh
       - if [ "$CODEBUILD_BUILD_SUCCEEDING" ] && [ "$CODE_COV_TOKEN" ]; then bash ./codecov.sh -t $CODE_COV_TOKEN -F uitest || true; fi
+
 artifacts:
   files:
-    - guitest.log
-    - jetbrains-core-gui/build/idea-sandbox/system/log/**/*
\ No newline at end of file
+    - "**/*"
+  base-directory: /tmp/testArtifacts
```

Comment:
nitpick: base directory is above files which looks weird

Reply:
Switched order